sources/commands/option_env.c

```diff
@@ -6,19 +6,20 @@
 /*   By: inikulin <inikulin@student.42berlin.de>    +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/09/30 00:08:59 by inikulin          #+#    #+#             */
-/*   Updated: 2024/10/12 01:11:40 by inikulin         ###   ########.fr       */
+/*   Updated: 2024/10/22 22:59:24 by inikulin         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "commands.h"
 
+// return value is ignored
 int	option_env(t_control control, t_treenode *node, t_param *param)
```

Comment:
what does name mean?

Reply:
That's a name of a bash built-in command we have to implement. Presumably "environment". try `man env`